packages/web-pixel-ui-extensions/src/extension-points/web-pixel-api.ts

```diff
@@ -0,0 +1,23 @@
+export type Version = 'unstable';
+
+/**
+ * The following APIs are provided to the Pixel extension point.
+ */
+export interface WebPixelApi {
+  analytics: {
+    subscribe(event: string, callback: (data: any) => void): void;
+  };
+  config: Record<string, unknown>;
```

Comment:
Nit: we called this `configuration` in checkout extensions

Reply:
@sorta-shopify Should we make that change or would it put us back too much given the different places that needs to be updated!?

Reply:
I think it may not be too bad to change, and now might be the time do it since I have open PRs for related stuff. I would lean towards making this change now I think